accepted/future-releases/static-extension-methods/feature-specification.md

```diff
@@ -461,6 +461,57 @@ That is: We do *not* allow implicit tear-off of an extension `call` method in a
+
+A library which is at a language versions prior to the release of static extension methods will not be able to use extension members:
+
+- It cannot declare an extension.
+- it cannot refer to an imported extension.
+- It cannot invoke an imported extension member.
+- It *can* re-export an extension from another library.
+
+A library which has not enabled static extension members cannot use the new syntax. It also cannot use the *override* syntax (`MyExt(o).member()`) even though it is grammatically valid as a function or constructor invocation. The extension is neither a class nor a function.
+
+If such a library imports an extension declaration, say `MyExt`, then any reference to that imported name is a compile-time error, the same way as accessing a name-conflicting import. The imported declaration is still there, and can cause naming conflicts, but attempting to use it is disallowed.
+
+Invocations which would otherwise check for extension members, do not. It is as if there are no extensions in scope, even if some were imported.
+
+The library can export any other library, and will do so blindly without needing to understand the exported declarations. The exporting library can still cause a naming conflict if it exports something else with the same name as an exported extension.
+
+*This is not the only possible option. It might be possible to enable use of extensions in libraries which cannot declare them. However, it would be only half a feature without the syntax for extension member override, and enabling that syntax would also be inconsistent. As such, the simplest and safest approach is to _disable_ extensions completely in legacy libraries. The cost of enabling extensions is trivial since it will merely be a matter of increasing the library SDK requirement. There is no migration needed for a non-breaking change.*
+
+## Interaction With Potential Future Features
+
+### Non-Null by Default
+
+The interaction with NNBD was discussed above. It will be possible to declare extensions on nullable and non-nullable types, and on a nullable type, `this` may be bound to `null`.
+
+### Sealed Classes
+
+If we introduce sealed classes, we may want to consider whether to allow extensions on sealed classes, since adding members even to a sealed class could still be a breaking change.
+
+One of the reasons for having sealed classes is that it ensures the author can add to the interface without breaking code. If adding a member changes the meaning of code which currently calls an extension member, that reason is eliminated. 
+
+Since it's possible to add extensions on superclass (including `Object`), it would not be sufficient to disallow *declaring* extensions on a sealed class, you would have to disallow *invoking* an extension on a sealed class, at least without an explicit override (which would also prevent breaking if a similarly named instance member is added).
+
+### Extension Types
```

Comment:
This section will be pretty much entirely opaque to anyone not on the language team.  If you keep it here, consider at least linking to the extension type proposal for context.

Reply:
I'll just remove it.